core/storage/feedback/gae_models.py

```diff
@@ -88,18 +97,42 @@ def generate_new_thread_id(cls, exploration_id):
 
     @classmethod
     def generate_full_thread_id(cls, exploration_id, thread_id):
+        """Generates a full thread ID which is unique globally.
```

Comment:
I don't think this is the point. It just generates a full thread id, given the exploration id and the thread id.